data/virt_autotest/host_unattended_installation_files/autoyast/dev_host_12.xml.ep

```diff
@@ -110,10 +110,14 @@
         <bridge>yes</bridge>
         <bridge_forwarddelay>15</bridge_forwarddelay>
         <!-- Use specified interface instead of a list to avoid ip lease inconsistency. Please refer to poo#135641. -->
-        % if ($check_var->('SUT_NETDEVICE', 'eth1') or $check_var->('SUT_NETDEVICE', 'em1')) {
+	% if ($get_var->('SUT_NETDEVICE')) {
```

Comment:
@Julie-CAO Thanks for the fix! Code LGTM. It would be even better if a comment can be added here to explain how it is supposed to be set for bridge ports.

Reply:
done